README.md

```diff
@@ -58,7 +58,7 @@ documented in [Embedding LLVM in your project](https://llvm.org/docs/CMake.html#
 Commands to checkout sources and build:
 ```bash
 cd <workspace>
-git clone https://github.com/intel/opencl-clang.git
+git clone https://github.com/intel/opencl-clang.git -b ocl-open-110
```

Comment:
Same as above.
Suggest to keep it as is?

Reply:
Original command is to checkout master. my command is to check ocl-open-110 branch